### ISSUE TITLE
.netstandard library - ##[error]Cannot read property 'textContent' of undefined

### ISSUE THREAD

**Issue by tmarkovski** [MrTarantula/ProjectVersionTools#5]
I'm getting the error `##[error]Cannot read property 'textContent' of undefined` when trying to read version from this project file. Here are the project specs.

```
<Project Sdk="MSBuild.Sdk.Extras/1.6.46">

  <PropertyGroup>
    <TargetFrameworks>netstandard2.0;xamarinios10</TargetFrameworks>
    <Version>1.1.0</Version>
    <AssemblyVersion>1.1.0</AssemblyVersion>
    <FileVersion>1.1.0</FileVersion>
  </PropertyGroup>
```

Does the plugin work only when "ApplicationVersion" is specified?

**Comment by MrTarantula** [comment]
I'm guessing you're using the "Set Project Version" task. Yes, [this line](https://github.com/MrTarantula/ProjectVersionTools/blob/386d5540b03b5d9085bb5f4a78558a3f09f3d38f/setprojectversion/setprojectversion.ts#L35) specifies that it works only for `ApplicationVersion`. This addon is meant for the old csproj/vbproj files. I haven't found time to update it for the new files.

**Comment by tmarkovski** [comment]
I'm using the "Get Project Version", noticed it only reads ApplicationVersion. I could submit a PR that adds a bit more configuration as to which field to use to read this. Would you be open to that?

**Comment by tmarkovski** [comment]
Here you go.

https://github.com/MrTarantula/ProjectVersionTools/pull/6

**Comment by MrTarantula** [comment]
Thanks. I'll merge when I have time to update the extension in the Marketplace.

**Comment by MrTarantula** [comment]
This has been merged and the extension is now v1.2.0. Thanks for your contribution!
